### PR TITLE
STAC-10098: kubernetes client fix

### PR DESCRIPTION
**Pull request by hierynomus**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,18 +3,21 @@ module github.com/stackvista/sandbox-operator
 go 1.13
 
 require (
+	cloud.google.com/go v0.53.0 // indirect
 	github.com/benbjohnson/clock v1.0.3
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/butonic/zerologr v0.0.0-20191210074216-d798ee237d84
-	github.com/caspr-io/mu-kit v0.0.101
```
Comment (vpartington) [inline go.mod:9]: Ha, where did that come from? Copy-paste? ;-)

File: internal/reaper/reaper.go [hunk 2]
```diff
@@ -35,14 +36,30 @@ type Reaper struct {
 
 func NewReaper(ctx context.Context, config *Config, notifier notification.Notifier) (*Reaper, error) {
 	logger := log.Ctx(ctx)
-	k8s, err := kubernetes.ConnectToKubernetes()
+
+	cfg, err := rest.InClusterConfig()
```
Comment (vpartington) [inline internal/reaper/reaper.go:40]: Hey, I'm remember that code. ;-)

**Review by vpartington — APPROVED** [review]
LGTM!

File: go.mod (hunk 1 above)
Comment (hierynomus) [reply go.mod:9]: Nope, it was a dep actually ;)

File: internal/reaper/reaper.go (hunk 2 above)
Comment (hierynomus) [reply internal/reaper/reaper.go:40]: Yes, somehow the dep didn't work as expected, so I had to write it mysefl.

**Pull request merged**